VocaDbWeb/Views/Event/EventsByDate.cshtml

```diff
@@ -15,6 +15,9 @@
 	<li>
 		@Html.ActionLink(Res.ViewBySeries, "EventsBySeries")
 	</li>
+	<li>
+		@Html.ActionLink(Res.ViewByVenues, "EventsByVenues")

```

Comment:
Actually, I would prefer getting rid of all these old ASP.NET pages. The search page and the "Upcoming" events page should be enough. I guess the EventsByVenues page can be there now that it's added, but I think it could be removed later.